styles.css

```diff
@@ -79,3 +91,204 @@
   line-height: 24px;
   color: #7f8cff;
 }
+
+/* Works Design */
+#portfolio {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 84px 24px 24px 24px;
+  gap: 110px;
+}
```

Comment:
- [ ] Your portfolio section is great and looks very similar to the Figma template, but you missed the requirement of using the `grid` in the main container. Kindly change that to match the requirements.
 - 
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/61919617/212411364-98c5075d-7780-4cfd-9575-bc8559f6f46b.png">